src/SDKs/TrafficManager/Management.TrafficManager/generate.cmd

```diff
@@ -4,7 +4,7 @@
 ::
 
 @echo off
-set autoRestVersion=0.16.0
+set autoRestVersion=0.13.2
```

Comment:
version: 0.13.2 is the latest, as per https://www.npmjs.com/package/autorest; I installed it as per instructions here https://github.com/Azure/autorest, using npm install -g autorest; it's the versioning that's all over the place (https://www.nuget.org/packages/AutoRest), but I have used the latest version; 

commit: I am waiting for @alvadb to merge https://github.com/Azure/azure-rest-api-specs/pull/1180